infrastructure/lambda/template.yaml

```diff
@@ -140,13 +140,13 @@ Mappings:
       integration: MONTHS
       production: MONTHS
 
-  useLegacyMapping:
+  dvaDirectEnabledMapping:
     Environment:
-      dev: "true"
-      build: "true"
-      staging: "true"
-      integration: "true"
-      production: "true"
+      dev: "false"
```

Comment:
Can we make this true in dev, build and staging so we can easily test

Reply:
Corrected.